v4l2-request-test.h

```diff
@@ -23,7 +23,12 @@
 #include <linux/types.h>
 #include <linux/v4l2-controls.h>
 #include <linux/videodev2.h>
-#include <h264-ctrls.h>
+#include <linux/media.h>
```

Comment:
Is there an issue when this headers is missing? It should be included directly in each file that needs it.

Reply:
Will see. Working to include your remarks and compile again.
So you mean, each source needing defines defined in media.h should include <linux/media.h> in the relevant source itself?

Reply:
Yes indeed. That's how I initially wrote it: include only the needed headers in each file. So any header in v4l2-request-test.h has to be there because a declaration in the header itself needs it.